contents/docs/new-to-posthog/switch-guide/migration-planning.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: What to expect as you migrate
+---
+
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+With your team onboarded and a successful pilot completed, you can turn your attention to **planning your migration**.
+
+***
+
+## Technical planning
+
+PostHog provides tools and guides to support you making this switch. Check out our [migration docs](/docs/migrate) for details on moving from [Amplitude](/docs/migrate/migrate-from-amplitude), [Mixpanel](/docs/migrate/mixpanel), [LaunchDarkly](/docs/migrate/launchdarkly) and others.
+
+Stumped on any detail of the process? If you’ve already signed up and started sending events with a forecasted bill of $500 or more, our **onboarding** team will reach out to help. If you haven’t heard from us yet, [drop us a line](/talk-to-a-human) with your migration questions!
+
+<CalloutBox icon="IconWarning" title="Avoid common setup pitfalls" type="caution">
+
+Don’t create separate projects for mobile and web. PostHog works best when you can see user behavior across all versions of your products in one set of data and views.
+
+Healthy integrations have a few things in common. It can help to review the [health check](/handbook/growth/sales/health-checks) documentation used by our sales team to tackle the most common ways to keep your bill manageable and predictable.
+
+If your historical data is above 100 GB or 200 million events, you should [reach out to our sales team](/talk-to-a-human) or [open a support ticket](/docs/support-options) so we can make sure everything imports smoothly.
+
+</CalloutBox>
+
+### Reducing disruption for your teammates
+
+New software can take a moment to click. One way to help your teammates get comfortable is making sure vital numbers are already waiting in your new PostHog setup.
+
+To support this, it can help to make a spreadsheet for people to request their most essential metrics. Use it to create insights in PostHog, then link them in the spreadsheet.
+
+You want to know what data your teammates need to keep making good decisions, and ensure those metrics are easy to find in PostHog. Capturing these requirements can also help you check that your technical implementation is capturing all necessary events and properties.
+
+While moving historical events over is straightforward, you’ll need plan for some time rebuilding the views and dashboards your team relies on.
+
+### Conversion tracking
+
+An urgent need from marketing and product teams is **tracking conversions**. Getting clear on how to do this in PostHog can reduce frustration as everyone settles in. These docs on using **[funnels](/docs/product-analytics/funnels)** are a great starting point for this kind of reporting.
+
+***
+
+## Providing demos
```

Comment:
Not important for shipping V1, but I think "How to demo to your teammates" might better as it's own page. More than half this is page is about this, rather than the nuts and bolts of migrating. It get they're related, but as we build this out I think we can split them up.